src/join-the-committee.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Join the project selection committee
+---
+
+# Join the project selection committee! <a name="join-the-project-selection-committee"></a>
+
+Thanks for being interested in joining the committee. The central task of the
+project selection committee is, of course, selecting projects; but more
+broadly, it's a way to get involved in the overall Fix Planet Club organisation.
+
+All our important documents are [public on
+GitHub](https://github.com/FixPlanet/org).
+
+## Expectations
+
+- **Project review** - Willingness, ability and time to review projects and participate
+in project interviews. Estimated time committment: 4-5 hours per month.
+- **Community involvement** - Willingness to join our [slack
+community](https://join.slack.com/t/fixplanetclub/shared_invite/zt-1dwnqqvuc-5sXa4HYsl81T9N35W~M9CA) and involve yourself in discussions that pique your interest.
+- **Internal governance** - Interest in participating in our general
+governance process, by participating in discussions on slack and/or GitHub
+around the overall project structure and operating procedures. Ultimately,
+responsibility for these decisions sits with the [trustees](/committee.html), but pragmatically 
+we believe that they should be established and discussed openly as a community.
+- **Term** - The expectation is that you will be on the committee for at least 1 year.
+- **Advocate** - A willingness to advocate for the program within your
+communities. A big part of the way we hope to reach diverse groups of people
+is through our committee members.
+
+## Criteria
+
+You don't need to be an expert to join this group. The essential conditions
+are:
+
+1. You (roughly[^agree]) agree with our [operating principles and core mission](https://github.com/FixPlanet/org#mission),
+2. You bring a perspective, set of personal experiences, local knowledge
+   and expertise that isn't already well-covered by our [existing committee
+   members](/committee.html). We value diversity, and are working hard to ensure this is
+   represented in our committee make-up,
+3. You have a passion for _somehow_ bringing technology, and critical (re-)thinking
```

Comment:
Maybe it's because I'm not a native english native speaker, but this sentence does not read extremely clear to me

Reply:
Ah, okay, I'm trying to say something like this:

> You are passionate about using technology, and critical re-thinking _of_ technology roles and applications, to the fight against the climate and associated crises.

I'm trying to say something like "yeah, it's great if you want to build a library, but if you also want to write an article where you show how AI is extremely harmful, or any optimisation is just going to make things worse, that's good too".

I mean, hopefully what people do would be more nuanced than that, but the main point is that it just have to _just_ be "additive" with some new library, framework, etc; it can also engage critically.

Does that make sense? Feel free to re-word however you see fit, if you have ideas, otherwise I'll think about it during the day.